index.html

```diff
@@ -76,50 +80,55 @@ <h2>Multi-Post
 				<li><a href="">javascript</a></li>
 			</ul>
 
-			<button class="btn">See Project</button>
+			<button type="button" class="btn">See Project</button>
+		</div>
 		</div>
 
-		<div>
-			<img src="/img/tonic2.svg" alt="">
-			<h2>TONIC</h2>
-			<ul class="work-exp">
-				<li>Canopy</li>
-				<li><img src="/img/circle.svg" alt=""></li>
-				<li>Back End Dev</li>
-				<li><img src="/img/circle.svg" alt=""></li>
-				<li>2015</li>
-			</ul>
-			<p>A daily selection of privately personalized reads; 
-				no accounts or sign-ups required.</p>
-      <ul class="lang">
-				<li><a href="">html</a></li>
-				<li><a href="">css</a></li>
-				<li><a href="">javascript</a></li>
-			</ul>
+		<div id="work_3">
+			<div class="img"><img src="/img/tonic2.svg" alt=""></div>
+		    <div class="content_work">
+				<h2>TONIC</h2>
+				<ul class="work-exp">
+					<li>Canopy</li>
+					<li><img src="/img/circle.svg" alt=""></li>
+					<li>Back End Dev</li>
+					<li><img src="/img/circle.svg" alt=""></li>
+					<li>2015</li>
+				</ul>
+				<p>A daily selection of privately personalized reads; 
```

Comment:
- [ ] [**OPTIONAL**] Kindly make sure that these project images shows perfectly, it will make your project more professional :+1: 

![Screenshot from 2023-01-21 01-58-12](https://user-images.githubusercontent.com/80458199/213827662-0b6bd903-794a-4a79-b36b-732bdac5cfc3.png)